cmd/snap-bootstrap/bootstrap/bootstrap.go

```diff
@@ -40,6 +42,12 @@ type Options struct {
 	KeyFile string
 	// RecoveryKeyFile is the location where the recovery key is written to
 	RecoveryKeyFile string
+	// LockoutAuthFile is the location where the lockout authorization is written to
+	LockoutAuthFile string
+	// AuthUpdateFile is the location where the authorization policy update data is written to
+	AuthUpdateFile string
```

Comment:
secboot itself always refers to this as policyUpdate,  let's call this PolicyUpdateFile  and --policy-update-file etc

Reply:
that's fine, I actually propose to call the actual file policy-update-data so the lengthy option seems fine